core/src/main/scala/com/raphtory/api/analysis/visitor/InterlayerEdge.scala

```diff
@@ -66,6 +66,7 @@ case class InterlayerEdge(
   override def containsState(key: String, includeProperties: Boolean): Boolean = ???
   override def getOrSetState[T](key: String, value: T, includeProperties: Boolean): T = ???
   override def appendToState[T: ClassTag](key: String, value: T): Unit = ???
+  override def clearState(key: String): Unit = ???
```

Comment:
This class needs some serious love xD

Reply:
agree haha, it's more that I think we'd need a discussion on what each of these things (mainly getState) would mean for an interlayer edge

Reply:
I think this is on @narnolddd